docs/authentication/3id-did/method.md

```diff
@@ -0,0 +1,55 @@
+# 3ID DID Method
+
+The [3ID DID Method (CIP-79)](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-79/CIP-79.md) is a [DID method](../../learn/glossary.md#did-methods) which can be used to [authenticate](../../build/authentication.md) to Ceramic and perform [writes](../../build/writes.md) to streams that rely on DIDs. 3ID DID is a powerful DID method that supports multiple keys, key rotations, and revocations. The 3ID DID Method is on the W3C's official DID method registry and is fully compliant with decentralized identity standards.
+
+## **Considerations**
+
+**Mutable DID document**: The 3ID DID Method is fully implemented on top of Ceramic. The primary benefit of the 3ID DID method is that it has a **mutable DID document**. The DID Document for 3ID DIDs is stored in a [TileDocument StreamType](../../streamtypes/tile-document/overview.md). This is enables 3ID DIDs to handle multiple keys, secure key rotations, and the addition and removal or arbitrary data.
```

Comment:
I feel like secure key rotations is noteworthy enough to be a top-level bullet.  I think that's one of the truly revolutionary things about 3ID over other DID methods

Reply:
made this change